### PR TITLE
Mutation to update payment data 

### PR DESCRIPTION
**Pull request by pedromtec**
#### What problem is this solving?

`setOrderPayment` mutation to update the payment data.

[Related story](https://app.clubhouse.io/vtex/story/34723/adicionar-etapa-de-parcelamento-no-checkout-payment)

#### How should this be manually tested?

Add some products to the cart and go to payment step.
[Workspace](https://paymentio--checkoutio.myvtex.com/)

#### Checklist/Reminders

- [ ] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️| New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- `SetOrderPayment` mutation to update payment data
```
Comment (lucasecdb) [inline CHANGELOG.md]: can you remove these empty spaces around the titles? we found out it makes rebasing more complicated with unnecessary conflicts in the changelog. you can remove the spaces of the `0.1.0` release as well
```suggestion
## [Unreleased]
### Added
- `SetOrderPayment` mutation to update payment data
```

Comment (pedromtec) [reply CHANGELOG.md]: It makes sense 💯

**Review by lucasecdb — APPROVED** [review]

**Review by marcoskwkm — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,9 +6,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `SetOrderPayment` mutation to update payment data
```
Comment (darlenedms) [inline CHANGELOG.md]:
```suggestion
- `setOrderPayment` mutation to update payment data
```

File: react/OrderPayment.tsx [hunk 3]
```diff
@@ -52,8 +60,18 @@ interface Status {
   value: TokenizedCard[] | string
 }
 
+interface UpdateOrderFormPaymentMutation {
+  updateOrderFormPayment: OrderForm
+}
+
+interface UpdateOrderFormPaymentMutationVariables {
+  paymentData: PaymentDataInput
+}
```
Comment (darlenedms) [inline react/OrderPayment.tsx:69]: Maybe add an empty line here.

**Review by darlenedms — APPROVED** [review]

**Pull request merged**